### ISSUE TITLE
[BUG]: False positive of no accessible name for img with alt text

### ISSUE THREAD

**Issue by YoshiJL** [IBMa/equal-access#2003]
### Project

accessibility-checker-engine, extension-checker

### Browser

Chrome, Firefox

### Operating system

Windows

### Description

The issue `Element <img> with "img" role has no accessible name` appears for `img` elements that have `alt` attribute filled. 

### Steps to reproduce

1. Perform a scan. 
2. Go to any `Element <img> with "img" role has no accessible name` recommendation. 
3. Find that the `img` element that is identified includes `alt` attribute.

**Comment by tombrunet** [comment]
This has already been fixed in #1980. This will deploy in the next release.
